tools/phenomenal/w4m/update-tools

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Thanks for this Pierrick, looks very good. Only one minor request/suggestion:

Where/when do you intend to run this update-tools script? Within the container? The container doesn't have docker itself and won't do docker-within-docker. If this does not need to be run inside the container itself, can we put this script outside of the tools folder, which gets copied entirely into the docker container?

Thanks!